### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by Dassderdie**
* Updates all packages (especially angular to 14)
* npm 8 is now required
* Fixes the `launch.json` for vscode
* Fixes the eslint rule that enforces `lodash-es` instead of `lodash`

### CODE REVIEW

**Comment by ClFeSc** [comment]
@Dassderdie I've reverted Cypress CI to Node 16 to let the pipeline pass and created #503 for it. This would allow us to continue working on other features/fixes. When you're happy with this you can merge, I'll approve.

**Review by ClFeSc — DISMISSED** [review]
Approved with reference to #503

**Comment by Dassderdie** [comment]
@ClFeSc I was able to reproduce vscode not hitting breakpoints in the frontend on my laptop.

This is the issue: https://github.com/angular/angular-cli/issues/23383

I, therefore, propose waiting for the LTS release of NodeJs 18 in October this year. -> reverting the changes here and closing #503 .

**Comment by ClFeSc** [comment]
What changes do you want to revert? All of them? Or just Node 18?

**Review by ClFeSc — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This project is currently developed as a [bachelor project](https://hpi.de/en/st
 ## Installation
 
 1. Make sure to have [git lfs](https://git-lfs.github.com/) installed.
-2. Install [NodeJs](https://nodejs.org/) (at least version 16.x) (if you need different node versions on your machine we recommend [nvm](https://github.com/nvm-sh/nvm) or [nvm for windows](https://github.com/coreybutler/nvm-windows))
+2. Install [NodeJs](https://nodejs.org/) (at least version 18.x) (if you need different node versions on your machine we recommend [nvm](https://github.com/nvm-sh/nvm) or [nvm for windows](https://github.com/coreybutler/nvm-windows))
```
Comment (Dassderdie) [inline README.md:33]:
```suggestion
2. Install [NodeJs](https://nodejs.org/) (at least version 16.x) (if you need different node versions on your machine we recommend [nvm](https://github.com/nvm-sh/nvm) or [nvm for windows](https://github.com/coreybutler/nvm-windows))
```

**Pull request merged**